assets/extensions/main.js

```diff
@@ -12,20 +12,23 @@ import { __ } from '@wordpress/i18n';
 import Header from './header';
 import Tabs from './tabs';
 import UpdateNotification from './update-notification';
+import WCCOMConnection from './update-notification/wccom-connection';
 import QueryStringRouter, { Route } from '../shared/query-string-router';
 import AllExtensions from './all-extensions';
 import FilteredExtensions from './filtered-extensions';
 import { Grid, Col } from './grid';
 
 const Main = () => {
 	const [ extensions, setExtensions ] = useState( false );
+	const [ isConnected, setIsConnected ] = useState( false );
 
 	useEffect( () => {
 		apiFetch( {
 			path: '/sensei-internal/v1/sensei-extensions?type=plugin',
 		} )
 			.then( ( result ) => {
 				setExtensions( result.extensions || [] );
+				setIsConnected( result.wccom_connected );
```

Comment:
I think we could rebase this PR temporarily to https://github.com/Automattic/sensei/pull/4214, because there are some changes that can impact here. The fetch now is in a store, so I think we should migrate this logic to there.

Reply:
I think I'll just wait until it's merged to `master` and then do any necessary refactoring.